apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-modal/styles.scss

```diff
@@ -1,10 +1,10 @@
-@import '@wordpress/base-styles/mixins';
-@import '@wordpress/base-styles/variables';
-@import '@wordpress/base-styles/breakpoints';
-@import '@wordpress/base-styles/z-index';
-@import '@automattic/typography/styles/variables';
-@import '@automattic/onboarding/styles/variables';
-@import '@automattic/onboarding/styles/mixins';
+@use '@wordpress/base-styles/mixins';
+@use '@wordpress/base-styles/variables' as variables3;
```

Comment:
What's the rationale between renaming `styles/mixins` to `styles-mixins` and then, not applying the same logic to `base-styles/variables`?

I know it's auto-generated by the tool, but still, that intrigues me 😂 

Could we rename those variables to something more explicit? There are usages of `variables3` inside the code and I have no idea where they come from.